web/themes/custom/subtheme_videogame/src/components/field/_mediaquery.scss

```diff
@@ -0,0 +1,46 @@
+@media screen and (max-width: 576px) {
```

Comment:
Please use the `$grid-breakpoints` variable for retrieving the media query values.